<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Editor</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="editor">
         <property name="documentTitle">
          <string/>
         </property>
         <property name="tabStopWidth">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_File"/>
   <addaction name="actionOpen_file"/>
   <addaction name="actionSave_File"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="separator"/>
   <addaction name="actionDownload"/>
   <addaction name="actionGet_RAM"/>
   <addaction name="actionGet_ROM"/>
   <addaction name="actionSave_RAM_to_ROM"/>
   <addaction name="separator"/>
   <addaction name="actionUse_RAM"/>
   <addaction name="actionUse_ROM"/>
   <addaction name="separator"/>
   <addaction name="actionDebug"/>
  </widget>
  <widget class="QDockWidget" name="BoardButtons">
   <property name="windowTitle">
    <string>Board buttons</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="SizerButtons">
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Terminal</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPlainTextEdit" name="terminal">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="tabStopWidth">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="butClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonEnd">
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="inputTerminal">
       <property name="placeholderText">
        <string>Input</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5">
   <property name="windowTitle">
    <string>Connection</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="comboPort"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSave_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Actions-document-save-icon.png</normaloff>:/icons/Actions-document-save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Actions-document-new-icon.png</normaloff>:/icons/Actions-document-new-icon.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Actions-document-save-as-icon.png</normaloff>:/icons/Actions-document-save-as-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Actions-document-open-icon.png</normaloff>:/icons/Actions-document-open-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Actions-network-connect-icon.png</normaloff>:/icons/Actions-network-connect-icon.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Button-Download-icon.png</normaloff>:/icons/Button-Download-icon.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionGet_ROM">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Folders-Uploads-Folder-icon.png</normaloff>:/icons/Folders-Uploads-Folder-icon.png</iconset>
   </property>
   <property name="text">
    <string>Get ROM</string>
   </property>
  </action>
  <action name="actionGet_RAM">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Button-Upload-icon.png</normaloff>:/icons/Button-Upload-icon.png</iconset>
   </property>
   <property name="text">
    <string>Get RAM</string>
   </property>
  </action>
  <action name="actionSave_RAM_to_ROM">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/CD-Burn-icon.png</normaloff>:/icons/CD-Burn-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save RAM to ROM</string>
   </property>
  </action>
  <action name="actionUse_RAM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Drives-RAM-Metro-icon.png</normaloff>:/icons/Drives-RAM-Metro-icon.png</iconset>
   </property>
   <property name="text">
    <string>Use RAM</string>
   </property>
  </action>
  <action name="actionUse_ROM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rom-manager-2-icon.png</normaloff>:/icons/rom-manager-2-icon.png</iconset>
   </property>
   <property name="text">
    <string>Use ROM</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Bug-icon.png</normaloff>:/icons/Bug-icon.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="toolTip">
    <string>Debug</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>butClear</sender>
   <signal>clicked()</signal>
   <receiver>terminal</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
